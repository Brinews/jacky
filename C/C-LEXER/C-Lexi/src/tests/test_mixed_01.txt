
1 
1111 11111222.34  
hello 1x08 
.,;() [] < > <> >= <= = : 
IF WHILE DO THEN PRINT 
+ - * / $
