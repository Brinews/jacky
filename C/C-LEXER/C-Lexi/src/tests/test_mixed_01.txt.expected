{1 , NUM , 2}
{1111 , NUM , 3}
{11111222.34 , REALNUM , 3}
{hello , ID , 4}
{1x08 , BASE08NUM , 4}
{ , DOT , 5}
{ , COMMA , 5}
{ , SEMICOLON , 5}
{ , LPAREN , 5}
{ , RPAREN , 5}
{ , LBRAC , 5}
{ , RBRAC , 5}
{ , LESS , 5}
{ , GREATER , 5}
{ , NOTEQUAL , 5}
{ , GTEQ , 5}
{ , LTEQ , 5}
{ , EQUAL , 5}
{ , COLON , 5}
{IF , IF , 6}
{WHILE , WHILE , 6}
{DO , DO , 6}
{THEN , THEN , 6}
{PRINT , PRINT , 6}
{ , PLUS , 7}
{ , MINUS , 7}
{ , MULT , 7}
{ , DIV , 7}
{ , ERROR , 7}
{ , END_OF_FILE , 9}
